index.js

```diff
@@ -9,8 +10,12 @@ exports.StatSerializer = Interface.StatSerializer;
 exports.ResourceSerializer = Interface.ResourceSerializer;
 exports.ResourceDeserializer = Interface.ResourceDeserializer;
 exports.Schemas = Interface.Schemas;
+exports.ResourcesRoute = Interface.ResourcesRoute;
```

Comment:
What is it for? I saw the PR for that: https://github.com/ForestAdmin/forest-express/pull/97/files

Reply:
From my app, I only have `forest-express-sequelize` and not `forest-express` so this is a a kind "proxy exports" :D Make sense?

Reply:
ok but why do you need this for you app?

Reply:
I've discovered an easiest way to implement hasmany relationships:

```
var express = require('express');
var router = express.Router();
var liana = require('forest-express-sequelize');

function orders(req, res, next) {
  req.query.filterType = 'and';
  req.query.filter = { merchant_id: req.params.merchantId };

  liana.ResourcesRoute.order.list(req, res, next);
}

router.get('/merchant/:merchantId/relationships/orders', orders);

module.exports = router;
```

Reply:
ok, why not, I didn't tested it.
But it is not related to the multiple connections database, right?